llvm/lib/Target/AMDGPU/AMDGPUAddGlobalForAtomicXor.cpp

```diff
@@ -0,0 +1,58 @@
+//===- AMDGPUAddGlobalForAtomicXor.cpp ------------------------------------===//
```

Comment:
I'm not sure if this pass should really live in the AMDGPU backend, it is so closely coupled with SYCL runtime that one could argue it should be contained within [SYCLLowerIR](https://github.com/intel/llvm/tree/sycl/llvm/lib/SYCLLowerIR).